src/wallet/bdb.cpp

```diff
@@ -738,27 +739,32 @@ void BerkeleyDatabase::ReloadDbEnv()
     }
 }
 
-Dbc* BerkeleyBatch::GetCursor()
+bool BerkeleyBatch::CreateCursor()
```

Comment:
I think `StartCursor` would be a better name, after all, the whole process (including details like whether something is created) is internal.

Reply:
Renamed to `StartCursor`.